engine/app/components/citizens_advice_components/page_review.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module CitizensAdviceComponents
+  class PageReview < Base
+    def initialize(page_review_date:, date_format: nil)
+      super
+      @date_format = date_format || "%d %B %Y"
+      @page_review_date = page_review_date
+    end
+
+    def call
+      content_tag(
+        :p,
+        t("citizens_advice_components.page_review.body_html", date: formatted_date),
+        class: "cads-page-review"
+      )
+    end
+
+    def formatted_date
+      I18n.l(@page_review_date, format: @date_format)
```

Comment:
💭 Thoughts wanted on this API. My preference is to have the component accept a date object, so we can define the default format. We previously required people to pass a pre-formatted string. Any strong feelings either way?

Reply:
Yeah I think this is fine - we could parse a UTC `Z` string or something but the component is always going to be in ruby land anyway so seems a bit overkill and we may as well use a date object 👍🏻